rotkehlchen/externalapis/etherscan.py

```diff
@@ -285,11 +339,34 @@ def get_transactions(
             for entry in result:
                 gevent.sleep(0)
                 try:
+                    # Handle genesis block transactions
+                    if entry['hash'].startswith('GENESIS'):
+                        if self.db is None:
+                            raise DeserializationError('DB not initialized')
+                        trace_id = self._latest_trace_id(self.db)
+                    else:
+                        trace_id = None
+
                     tx = deserialize_ethereum_transaction(  # type: ignore
                         data=entry,
                         internal=is_internal,
```

Comment:
Nope, we're doing this so that normal transactions won't break, we're doing this hacky thing of making an internal txn to 0x0 so that we can add the txn details to DB, previously it would be skipped.